### DISCUSSION TITLE
Using Cloudflare as a Reverse Proxy, but 'Invalid Configuration' error occurred.

### DISCUSSION THREAD

**Discussion by julia-light** [vercel/community#6214] [Help]
### Summary

 I've followed this guide [using-cloudflare-as-a-reverse-proxy](https://vercel.com/docs/integrations/cloudflare#using-cloudflare-as-a-reverse-proxy):
 1. create a CNAME record pointing to cname.vercel-dns.com.
 2. Enable the Cloudflare proxy to send traffic through Cloudflare first.
 3. Set Cloudflare SSL mode to Full

When I configuring a domain on the Vercel dashboard, it prompts me with '**Invalid Configuration**'

### Steps to Reproduce

Followed this guide [using-cloudflare-as-a-reverse-proxy](https://vercel.com/docs/integrations/cloudflare#using-cloudflare-as-a-reverse-proxy)

**Comment by igorlanko** [comment]
Have the same issue. Did you find a solution? Do you also have the A record created, or are you just using CNAME to send traffic through Cloudflare (orange cloud)?

**Reply by derek-wang** [reply]
got the same error, but the site works, i only have CNAME added

**Reply by igorlanko** [reply]
@derek-wang, I tried various combinations, but as long as it goes through the Cloudflare, it works, but Vercel still shows the error in the domains 🤷

**Reply by derek-wang** [reply]
yup, just little bit annoying seeing the error
